go.mod

```diff
@@ -3,12 +3,17 @@ module github.com/asecurityteam/runsqs
 go 1.13
 
 require (
+	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883 // indirect
```

Comment:
Does this lib have an explicit version we can build against ?

Reply:
If not available, feel free to mark this as resolved

Reply:
i'm not sure what exactly that is... that is generated from `go mod vendor`